src/pages/index.js

```diff
@@ -43,9 +48,12 @@ export async function getServerSideProps(context) {
     const { members } = await res.json();
 
     for (const memberData of members) {
+      const img_url = memberData.picture
+        ? getCloudinaryImgURL(memberData.picture.publicId, `w_${MAX_WIDTH}`)
+        : getImgURL(memberData.username, 'img.png');
```

Comment:
Can we add this behind the feature flag? meaning this feature should only work if `dev=true`